Extensions/PlatformBehavior/PlatformerObjectBehavior.cpp

```diff
@@ -53,6 +53,8 @@ PlatformerObjectBehavior::GetProperties(
       gd::String::From(behaviorContent.GetDoubleAttribute("jumpSpeed")));
```

Comment:
There is still an error in this file. Have you tried manually your changes and checked that you can change the climbing speed in the editor and that it works?

Reply:
Thanks for notifying :)